config.go

```diff
@@ -114,6 +115,13 @@ func unmarshalTypedConfig(cfg map[string]interface{}) error {
 
 // apply applies the configuration.
 func (c *srcfileConfig) apply() error {
+	// KLUDGE: determine whether we're in the stdlib and if so, set GOROOT to "." before applying config.
+	// This is necessary for the stdlib unit names to be correct.
+	cloneURL, _ := exec.Command("git", "config", "--get", "remote.origin.url").CombinedOutput()
```

Comment:
It's better to use `Output` rather than `CombinedOutput` here.

`CombinedOutput` will include `stderr`, which may include warnings or other things that get in the way.

Ideally, you should get `Output`, check that `err` is `nil`, and then use the value of `cloneURL`.

See https://github.com/sourcegraph/go-vcs/issues/30 for a full description of how using `CombinedOutput` for parsing git output may cause failures in some cases.